src/components/Placeholder/Placeholder.js

```diff
@@ -0,0 +1,25 @@
+// Import Styling
+import './Placeholder.scss';
+
+// Placeholder string
+const Placeholder = () => (`
```

Comment:
Any reason we're not making this a real react component rather than using a template string?

Reply:
Oh I see, because it's passed in `dangerouslySetInnerHTML`. We could key off the `contentLoaded` state and use a proper component. Once we use `remark-react` though we can avoid `dangerouslySetInnerHTML` entirely and this can definitely become a proper component.

Reply:
Yeah, that's the only reason, I actually crafted it initially as a component and had to roll back when I realized that. We can definitely refactor it when we introduce remark to the game.